compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -89,16 +88,22 @@ internal actual class ComposeWindow : UIViewController {
             val keyboardInfo = arg.userInfo!!["UIKeyboardFrameEndUserInfoKey"] as NSValue
             val keyboardHeight = keyboardInfo.CGRectValue().useContents { size.height }
             val screenHeight = UIScreen.mainScreen.bounds.useContents { size.height }
+            val magicMultiplier = density.density - 1 // todo magic number
```

Comment:
I found a bug with wrong conversions to global coordinates on newer iPhones.
This is temporary workaround that helps with. In the future, I will try to rewrite this part of the code.

Reply:
It works fine with density 3 and 2. But, may have problems with another value.